data/json/items/gun/223.json

```
@@ -390,12 +382,81 @@
     "modes": [ [ "DEFAULT", "semi-auto", 1 ] ]
   },
   {
+    "//": "Second group, simply groups the two M4 carbines together.",
+    "id": "m4_carbine",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "M4 Carbine" },
```

Item names are lowercase.
```suggestion
    "name": { "str": "M4 carbine" },
```